x/staking/simulation/genesis.go

```diff
@@ -78,8 +85,9 @@ func RandomizedGenState(simState *module.SimulationState) {
 		valAddrs[i] = valAddr
 
 		maxCommission := sdk.NewDecWithPrec(int64(simulation.RandIntBetween(simState.Rand, 1, 100)), 2)
+		curCommissionRate := simulation.RandomDecAmount(simState.Rand, maxCommission.Sub(minComRate)).Add(minComRate)
```

Comment:
Why did you make this change?

Reply:
I copied from v0.42.11-panacea branch (https://github.com/medibloc/cosmos-sdk/blob/4680505be37f589b16cfd9e48054c786a6b39a0e/x/staking/simulation/genesis.go#L88)

Reply:
Ah, that's right.
It is not important, but in the current test, if the value of min commission is not 0% (if it exceeds 1%), curCommissionRate may come out negative.
But as I mentioned above, it doesn't matter. :)